### DISCUSSION TITLE
When to release a new version in Nuget? I'm looking forward to use TLS as SCU...

### DISCUSSION THREAD

**Discussion by zhuthree** [fo-dicom/fo-dicom#1592] [General]
When to release a new version in Nuget? I'm looking forward to use TLS as SCU...

**Comment by mrbean-bremen** [comment]
This is a duplicate of  #1563, though that has not been answered yet. 

@gofal , @amoerie - I think it is time for planning a new release. It looks like there are a lot of breaking changes, and still a lot to finish, but I would suggest to offhold new features (like the support for enhanced mult-frames) for after a release.
I know that others are also waiting for the TLS feature.
What do you think?

**Comment by gofal** [comment]
I understand the need and will to a release before the multi-frame, propably this weekend.
There are enough breaking changes so that the users, who will update to the new release, for shure will check the code and the new version in detail.

**Comment by amoerie** [comment]
Release away! 🚀
